x-pack/plugins/cloud_security_posture/server/saved_objects/mappings.ts

```diff
@@ -46,6 +46,14 @@ export const cspRuleTemplateSavedObjectMapping: SavedObjectsTypeMappingDefinitio
     metadata: {
       type: 'object',
       properties: {
+        name: {
+          type: 'keyword',
+          fields: {
+            text: {
+              type: 'text',
+            },
+          },
+        },
```

Comment:
Yes - in order to use the search box on the rules page (the page that shows all the benchmark rules), we need to add the `name` field to the rule-template schema.
Otherwise, you cannot aggregate based on this field.